### PR TITLE
Update missing instruction in readme

### LINKED ISSUE #5: Update missing instruction in readme
**Is your feature request related to a problem? Please describe.**
- In order to work with academix-frontend you need set up the core from sef-core repo but that instructions are missing in the `README.md` file

**Describe the solution you'd like**
- Add the proper instructions to setup core in readme file.

**Are you working on this issue?**
Yes

### PR DESCRIPTION
**Pull request by anjula-sack**
## Purpose

- The purpose of this PR is to fix #5 

## Goals

- Update missing instruction in readme

## Approach

- Update missing instruction in `README.md`

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,57 @@
...
+3. Building the project
+    1. Build the project using maven
+       ```
+       mvn install
+       ```
+        Optional : You can build the modules separately
+        
+    2. Copy the generated `.war` file to the tomcat's webapp directory.  
+       ```
+       cp backend/target/core.war /path/to/tomcat/webapps
```
Comment (jayasanka-sack) [inline README.md:46]: frontend cp is missing

File: README.md [hunk 2]
```diff
@@ -1,14 +1,57 @@
...
+       ```
+### Building the frontend
```
Comment (jayasanka-sack) [inline README.md:48]: Building the Academix frontend

File: README.md [hunk 3]
```diff
@@ -1,14 +1,57 @@
 # SEF academix-frontend
 
+This build needs two modules
+1. Backend from sef-core
+    Contains the following APIs,
+    1. Multiverse Programme
+    
+2. Frontend from academix-frontend
+     Contains the academix home
+
```
Comment (jayasanka-sack) [inline README.md:10]: Do not mention this here.

Add Sef-core as a Prerequisites

File: README.md [hunk 4]
```diff
@@ -1,14 +1,57 @@
...
 
 ### Prerequisites
 1. Tomcat 9
 2. Maven
+3. MySQL 8
 
-### Building the project
-
-1. Build the project using maven
+### Building the backend
```
Comment (jayasanka-sack) [inline README.md:18]: Add a new heading named Building the SEF-Core or something like that.

And mention why we need this

File: README.md [hunk 5]
```diff
@@ -5,15 +5,24 @@
...
 1. Tomcat 9
 2. Maven
+3. MySQL 8
+4. sef-core
 
-### Building the project
-
-1. Build the project using maven
+### Building the SEF-Core
+- SEF-Core includes the main api's so you need to build it in order to build this project.
```
Comment (jayasanka-sack) [inline README.md:12]: Sef core includes APIs and the admin dashboard for the academic application.
In order to work with academix-frontend, you need to build sef-core and deploy it on tomcat (core.war and dashboard.war). 
Follow instructions in [sef-core](https://github.com/sef-global/sef-core/blob/master/Readme.md) to build it.

File: README.md [hunk 6]
```diff
@@ -5,15 +5,24 @@
...
+3. MySQL 8
+4. sef-core
 
-### Building the project
-
-1. Build the project using maven
+### Building the SEF-Core
+- SEF-Core includes the main api's so you need to build it in order to build this project.  
+1. Follow instructions in [sef-core](https://github.com/sef-global/sef-core/blob/master/Readme.md) to build the backend.
+### Building the academix frontend
```
Comment (jayasanka-sack) [inline README.md:14]: Add a space before the header

File: README.md (hunk 5 above)
Comment (anjula-sack) [reply README.md]: Thanks

File: README.md (hunk 6 above)
Comment (anjula-sack) [reply README.md:17]: ok

**Pull request merged**